Wiki/FAQ.md

```diff
@@ -0,0 +1,709 @@
+
+# Staking
+
+## How is the amount of stake-reward calculated?
+
+The stake reward is calculated by two combined
+algorithms.
+
+`Proof of Stake rewards will give you 10 GRC for each block you stack`  
+`Proof of Research rewards for BOINC contribution, though research activity. A PoR block is a combined payment 10 GRC and research reward.`
+
+## The wallet says "Not Staking because you don't have mature coins" , how long does it take for coins to mature?
+
+It takes about 16 hours for GRC to become mature from the time of
+arrival in your wallet. After a specific set of coins have Staked, they
+will need to mature again for 16 hours before they can Stake again.
+
+## Why can't I send coins in my wallet balance?
+
+You may have a balance reserved in stake. If this ever happens to you do
+this in the console:
+
+`reservebalance true AMOUNT`
+
+Where AMOUNT is the amount you want to send
+
+Wait a few blocks
+
+Then send
+it.
+
+## When I get an interest/mining payment, a portion of my GRC looks to be moving from my total balance into the "Stake" line of the wallet balance. Is that correct?
+
+Every 4 hours the wallet is on-line, it looks for old coins that can
+accrue interest and moves them from 'Balance' to 'Stake' then it works
+on staking. Once staked, it moves the staked balance back to
+balance.
+
+## I lose my stakeweight after receiving PoR? The "maturecoin-creationprocess" is starting again after each mined PoR?
+
+Yes that is normal, because your node builds up a set of stakeable coins
+and once you stake them the process takes about 4 more hours to get new
+mature coins, unless some have been found in between (it's a constant
+process).
+
+## Do I have to leave my wallet unencrypted to be able to stake?
+
+Well, if you encrypt your wallet and enter `walletpassphrase
+\[passphrase\] 86400`, that would unlock it for 24 hours for staking.
+
+Your wallet file remains encrypted and since you do not have to enable
+external RPC access for mining anymore, it's safe aside from physical
+access or perhaps memory leakage.
+
+## Can I stake with encrypted wallet?
+
+Yes, but you have to unlock it first. It's okay to unlock it "for
+staking only".
+
+The console walletpassphrase command does what clicking the lock button
+does.
+
+To lock Type
+
+`walletlock`
+
+to unlock for a certain time type
+
+`walletpassphrase PASSWORD TIMEOUT`
+
+to unlock type
+
+`walletpassphrase PASSWORD`
+
+to unlock only for staking type
+
+`walletpassphrase PASSWORD TIMEOUT true`
+
+## How do you turn staking on and off?
+
+`walletpassphrase "your passphrase in double quotes" timeout true`
+
+where timeout is a number of seconds, e.g. 10000 and true is a toggle
+that unlocks the wallet for staking
+only.
+
+# BOINC
+
+## Choosing BOINC projects: In BOINC, is there still a defined whitelist of "projects" or can we use any project now that has Gridcoin as a Team Name?
+
+Running "list projects" in console will list all the whitelisted

```

Comment:
```suggestion
Running "listprojects" in console will list all the whitelisted
```

Reply:
would just `projects` be better?

Output of listprojects:
```
{
"Amicable_Numbers": {
"display_name": "Amicable Numbers",
"url": "http://sech.me/boinc/Amicable/@",
"base_url": "http://sech.me/boinc/Amicable/",
"display_url": "http://sech.me/boinc/Amicable/",
"stats_url": "http://sech.me/boinc/Amicable/stats/",
"time": "2020-05-08 22:29:16 UTC"
},
"asteroids@home": {
"display_name": "asteroids@home",
"url": "http://asteroidsathome.net/boinc/@",
"base_url": "http://asteroidsathome.net/boinc/",
"display_url": "http://asteroidsathome.net/boinc/",
"stats_url": "http://asteroidsathome.net/boinc/stats/",
"time": "2020-05-08 20:44:17 UTC"
},
"collatz_conjecture": {
"display_name": "collatz conjecture",
"url": "http://boinc.thesonntags.com/collatz/@",
"base_url": "http://boinc.thesonntags.com/collatz/",
"display_url": "http://boinc.thesonntags.com/collatz/",
"stats_url": "http://boinc.thesonntags.com/collatz/stats/",
"time": "2020-05-08 22:29:38 UTC"
},
"cosmology@home": {
"display_name": "cosmology@home",
"url": "http://www.cosmologyathome.org/@",
"base_url": "http://www.cosmologyathome.org/",
"display_url": "http://www.cosmologyathome.org/",
"stats_url": "http://www.cosmologyathome.org/stats/",
"time": "2020-05-08 20:52:19 UTC"
},
"einstein@home": {
"display_name": "einstein@home",
"url": "http://einstein.phys.uwm.edu/@",
"base_url": "http://einstein.phys.uwm.edu/",
"display_url": "http://einstein.phys.uwm.edu/",
"stats_url": "http://einstein.phys.uwm.edu/stats/",
"time": "2020-05-08 22:30:02 UTC"
},
"gpugrid": {
"display_name": "gpugrid",
"url": "http://www.gpugrid.net/@",
"base_url": "http://www.gpugrid.net/",
"display_url": "http://www.gpugrid.net/",
"stats_url": "http://www.gpugrid.net/stats/",
"time": "2020-05-08 21:00:22 UTC"
},
"lhc@home_classic": {
"display_name": "lhc@home classic",
"url": "https://lhcathome.cern.ch/lhcathome/@",
"base_url": "https://lhcathome.cern.ch/lhcathome/",
"display_url": "https://lhcathome.cern.ch/lhcathome/",
"stats_url": "https://lhcathome.cern.ch/lhcathome/stats/",
"time": "2020-05-08 21:04:24 UTC"
},
"milkyway@home": {
"display_name": "milkyway@home",
"url": "http://milkyway.cs.rpi.edu/milkyway/@",
"base_url": "http://milkyway.cs.rpi.edu/milkyway/",
"display_url": "http://milkyway.cs.rpi.edu/milkyway/",
"stats_url": "http://milkyway.cs.rpi.edu/milkyway/stats/",
"time": "2020-05-08 21:08:26 UTC"
},
"nfs@home": {
"display_name": "nfs@home",
"url": "http://escatter11.fullerton.edu/nfs/@",
"base_url": "http://escatter11.fullerton.edu/nfs/",
"display_url": "http://escatter11.fullerton.edu/nfs/",
"stats_url": "http://escatter11.fullerton.edu/nfs/stats/",
"time": "2020-05-08 21:12:27 UTC"
},
"numberfields@home": {
"display_name": "numberfields@home",
"url": "https://numberfields.asu.edu/NumberFields/@",
"base_url": "https://numberfields.asu.edu/NumberFields/",
"display_url": "https://numberfields.asu.edu/NumberFields/",
"stats_url": "https://numberfields.asu.edu/NumberFields/stats/",
"time": "2020-05-08 22:31:24 UTC"
},
"odlk1": {
"display_name": "odlk1",
"url": "https://boinc.multi-pool.info/latinsquares/@",
"base_url": "https://boinc.multi-pool.info/latinsquares/",
"display_url": "https://boinc.multi-pool.info/latinsquares/",
"stats_url": "https://boinc.multi-pool.info/latinsquares/stats/",
"time": "2020-05-08 21:20:31 UTC"
},
"rosetta@home": {
"display_name": "rosetta@home",
"url": "http://boinc.bakerlab.org/rosetta/@",
"base_url": "http://boinc.bakerlab.org/rosetta/",
"display_url": "http://boinc.bakerlab.org/rosetta/",
"stats_url": "http://boinc.bakerlab.org/rosetta/stats/",
"time": "2020-05-08 21:24:33 UTC"
},
"SRBase": {
"display_name": "SRBase",
"url": "http://srbase.my-firewall.org/sr5/@",
"base_url": "http://srbase.my-firewall.org/sr5/",
"display_url": "http://srbase.my-firewall.org/sr5/",
"stats_url": "http://srbase.my-firewall.org/sr5/stats/",
"time": "2020-05-08 21:28:34 UTC"
},
"TN-Grid": {
"display_name": "TN-Grid",
"url": "http://gene.disi.unitn.it/test/@",
"base_url": "http://gene.disi.unitn.it/test/",
"display_url": "http://gene.disi.unitn.it/test/",
"stats_url": "http://gene.disi.unitn.it/test/stats/",
"time": "2020-05-08 21:32:35 UTC"
},
"universe@home": {
"display_name": "universe@home",
"url": "http://universeathome.pl/universe/@",
"base_url": "http://universeathome.pl/universe/",
"display_url": "http://universeathome.pl/universe/",
"stats_url": "http://universeathome.pl/universe/stats/",
"time": "2020-05-08 21:36:37 UTC"
},
"World_Community_Grid": {
"display_name": "World Community Grid",
"url": "https://www.worldcommunitygrid.org/boinc/@",
"base_url": "https://www.worldcommunitygrid.org/boinc/",
"display_url": "https://www.worldcommunitygrid.org/",
"stats_url": "https://www.worldcommunitygrid.org/boinc/stats/",
"time": "2020-05-08 22:31:44 UTC"
},
"yafu": {
"display_name": "yafu",
"url": "http://yafu.myfirewall.org/yafu/@",
"base_url": "http://yafu.myfirewall.org/yafu/",
"display_url": "http://yafu.myfirewall.org/yafu/",
"stats_url": "http://yafu.myfirewall.org/yafu/stats/",
"time": "2020-05-08 21:44:40 UTC"
},
"yoyo@home": {
"display_name": "yoyo@home",
"url": "http://www.rechenkraft.net/yoyo/@",
"base_url": "http://www.rechenkraft.net/yoyo/",
"display_url": "http://www.rechenkraft.net/yoyo/",
"stats_url": "http://www.rechenkraft.net/yoyo/stats/",
"time": "2020-05-08 21:48:41 UTC"
}
}
```
Output of projects:
```
[
{
"Project": "Amicable Numbers",
"URL": "http://sech.me/boinc/Amicable/"
},
{
"Project": "asteroids@home",
"URL": "http://asteroidsathome.net/boinc/"
},
{
"Project": "collatz conjecture",
"URL": "http://boinc.thesonntags.com/collatz/"
},
{
"Project": "cosmology@home",
"URL": "http://www.cosmologyathome.org/"
},
{
"Project": "einstein@home",
"URL": "http://einstein.phys.uwm.edu/"
},
{
"Project": "gpugrid",
"URL": "http://www.gpugrid.net/"
},
{
"Project": "lhc@home classic",
"URL": "https://lhcathome.cern.ch/lhcathome/"
},
{
"Project": "milkyway@home",
"URL": "http://milkyway.cs.rpi.edu/milkyway/"
},
{
"Project": "nfs@home",
"URL": "http://escatter11.fullerton.edu/nfs/"
},
{
"Project": "numberfields@home",
"URL": "https://numberfields.asu.edu/NumberFields/"
},
{
"Project": "odlk1",
"URL": "https://boinc.multi-pool.info/latinsquares/"
},
{
"Project": "rosetta@home",
"URL": "http://boinc.bakerlab.org/rosetta/"
},
{
"Project": "SRBase",
"URL": "http://srbase.my-firewall.org/sr5/"
},
{
"Project": "TN-Grid",
"URL": "http://gene.disi.unitn.it/test/"
},
{
"Project": "universe@home",
"URL": "http://universeathome.pl/universe/"
},
{
"Project": "World Community Grid",
"URL": "https://www.worldcommunitygrid.org/"
},
{
"Project": "yafu",
"URL": "http://yafu.myfirewall.org/yafu/"
},
{
"Project": "yoyo@home",
"URL": "http://www.rechenkraft.net/yoyo/"
}
]
```

Reply:
Yeah that would work